lib/nexmo_markdown_renderer/config/load_config.rb

```diff
@@ -2,8 +2,8 @@ module Nexmo
   module Markdown
     class Config
       def self.docs_base_path
-        @docs_base_path ||= (defined?(Rails) && Rails.application != nil && Rails.application.configuration.docs_base_path) || ENV.fetch('DOCS_BASE_PATH', '.')
+        @docs_base_path ||= ENV.fetch('DOCS_BASE_PATH', '.')
```

Comment:
Why do we no longer need to check if we're inside a Rails application?

Reply:
we could keep the check, although we would need to change how it is done.
We set the env variable in the rails app, so there isn't really a need to check if we are inside an app